apps/studio/pages/project/[ref]/auth/hooks.tsx

```diff
@@ -15,9 +20,21 @@ const Hooks: NextPageWithLayout = () => {
     return <NoPermission isFullPage resourceText="access your project's auth hooks" />
   } else {
     return (
-      <FormsContainer>
-        <HooksListing />
-      </FormsContainer>
+      <div>
+        <ScaffoldHeader className="pb-0">
+          <ScaffoldContainer id="auth-page-top">
+            <ScaffoldTitle>Auth Hooks</ScaffoldTitle>
```

Comment:
likewise - should the page title match the side menu label?
![image](https://github.com/user-attachments/assets/192fdc17-9f74-4d14-8393-03b3d2b0a07f)
![image](https://github.com/user-attachments/assets/e8ce67e6-97c3-436b-9339-6aec009f22cb)

Reply:
I would say yes.